### PR TITLE
Registration and CSS

### LINKED ISSUE #33: Provide Authentication / Login pages for public users
In order to submit talk proposals, users must first be authenticated, but there's currently no way for a non-admin user to do so.  Provide a standard set of authentication pages and appropriate menu links on the front end templates:
- Login
- Register
- Logout
- Change current password
- Reset / Forgot password

Should probably be dependent #19 being completed first.

### PR DESCRIPTION
**Pull request by jacques-cadet**
- Add registration configs to settings.py.
- Add registration url to podium/urls.py.
- Add login form and registration and
    admin link with permissions to nav.html
- Add registration information to README.md.
- Add registration information to requirements.txt.
- Copy registration templates into the project templates folder
    and update registration htmls with BS3 classes and forms.
- Create 'static' folder in project folder.
- Add static folder location to STATICFILES_DIRS in settings.py.
- Move contents of style.html to 'podium/static' folder
    as podium.css.
- Remove style.html.

Resolves #33

### CODE REVIEW

File: podium/static/poduim.css [hunk 1]
```diff
@@ -1,17 +1,44 @@
-<style type="text/css">
-	@import url('https://fonts.googleapis.com/css?family=Days+One|Jura|Marck+Script|Questrialv|Tangerine');
+@import url('https://fonts.googleapis.com/css?family=Jura|Questrial');
+
+	*{
```
Comment (jbonzo) [inline podium/static/poduim.css]: Can you explain the `*`?

File: podium/static/poduim.css [hunk 2]
```diff
@@ -52,21 +78,50 @@
...
 
+	/*NAVBAR UNDERLINE*/
+	.navbar-default .navbar-nav > li > a{
+		position: relative;
+		color: #005580;
+		font-family: 'Jura', sans-serif;
+		text-decoration: none;
+	}
+	.navbar-default .navbar-nav > li > a:after{
+		content: "";
```
Comment (jbonzo) [inline podium/static/poduim.css]: What does an empty string do?

File: templates/registration/logout.html [hunk 3]
```diff
@@ -0,0 +1,20 @@
+{% extends "registration/registration_base.html" %}
+{% load i18n %}
+
+{% block title %}{% trans "Logged out" %}{% endblock %}
+
+{% block content %}
+<div class="container">
```
Comment (jbonzo) [inline templates/registration/logout.html:7]: I noticed that you are using the `container` class but previous PR's use `container-fluid`, im not an expert in bootstrap so do you know the difference?

Comment (jacques-cadet) [reply templates/registration/logout.html:7]: In  CSS the * represents all elements, BS containers can be applied 2 ways, container - has margins on both sides and anything placed within it maintains the same. Container-fluid spans the total width of the document as will anything placed inside it.

**Review by jrrickerson — APPROVED** [review]
Looks good to me.  Thanks for the PR, and sorry for the long delay.
NOTE:  We'll need to add email settings so registration won't fail, but we can do this pretty quickly.

**Pull request merged**